info.md

```diff
@@ -99,7 +99,31 @@ re-execute notebooks.
 
 Plan: create an extra repository just for these `*ipynb` files, so that creating
 a container for Binder doesn't need to clone the big repository that contains
-all the html files.
+all the html files. This is realised with the next section:
+
+coronavirus-2020/tools/binder
+------------------------------
+
+-   repository <https://github.com/oscovida/binder> 
+
+-   this started (24 May 2020) as a copy of the coronavirus-2020/tools/wwwroot/ipynb
+    directory, but in a separate repository to make the binder start up time shorter
+    (because this repo is faster to clone than the massive wwwroot one.)
+    
+-   for now, we rsync everything from coronavirus-2020/tools/wwwroot/ipynb to
+    coronavirus-2020/tools/binder/ipynb
+    
+    In the long run, we could consider to write ipynb files directly into the
+    binder directory.
+
+-   The files committed to the webpages repository must contain the most recent
+    `coronavirus` in the `ipynb` subdirectory and `requirements.txt` and
+    `apt.txt` as those are needed by binder to execute the notebooks.
+
+-   By also commiting the `cachedir`, people don\'t need to re-fetch the data on
+    the binder service (fetching of the German data set varies between 1s and 60
+    seconds). The downside is that the data will grow stale over time.
+    (At the moment, 25May 2020, we don't copy the cachedir into the binder directory.)
```

Comment:
This approach is fine, but in the long run it'd lead to the repo growing in size a lot.

Another way to do this would be to pull the data for the cachedir in during the image build stage of the binder container, this way it's already there when binder starts, but doesn't have to be included in the git repository.

Reply:
[postBuild can be used for this](https://mybinder.readthedocs.io/en/latest/config_files.html#postbuild-run-code-after-installing-the-environment), it runs right before the container is finalised and uploaded.

Reply:
I am tempted to drop the cachedirectory from the binder repository as the time to fetch the data on the binder instances is fairly small (and consistently under 10 seconds even for the RKI data, often just 2 or 5 seconds; Johns Hopkins is nearly always around or below 2 seconds).